torch/quantization/ns/graph_matcher.py

```diff
@@ -280,18 +284,58 @@ def _get_node_relationship_type(
             return NodeTypeRelationship.NOT_RELATED
     return NodeTypeRelationship.NOT_RELATED
 
-def _get_name_for_subgraph_pair(
+def _get_name_for_subgraph(
     start_node_a: Node,
     end_node_a: Node,
-    start_node_b: Node,
-    end_node_b: Node,
+    gm_a: GraphModule,
```

Comment:
nit: Since there's no model b now, may just use name without "_a" like "gm"?

Reply:
sure, let me do that in a future PR (soon)